src/api/routes/auth/register.ts

```diff
@@ -225,6 +225,14 @@ router.post(
 		}
 
 		if (body.password) {
+                        if(body.password.length < register.password.minLength){
```

Comment:
```suggestion
                        const { minLength } = register.password; 
                        if (minLength && body.password.length < minLength) {
```

`register_password_minLength` might be null in config which would fail every password?

Reply:
how about using a default value like 8 if its null?

Reply:
works for me